### DISCUSSION TITLE
Add support for chaning tab size preference

### DISCUSSION THREAD

**Discussion by LitoMore** [community/community#133633] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.170.0 (136692889)
Device: iPhone 15 Pro (iOS 18.0)
</details>

The current GitHub mobile app does not support adjusting the tab size. The GitHub web already have this feature, please consider adding this feature to GitHub mobile as well.

Thanks.

**Comment by stevepopovich** [comment]
Hi @LitoMore thanks for the feedback. I passed this request on to the team. Thanks!
